src/NHSD.BuyingCatalogue.Ordering.Api/Logging/SerilogRequestLoggingOptions.cs

```diff
@@ -6,9 +6,9 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.Logging
 {
     public static class SerilogRequestLoggingOptions
     {
-        internal const string HealthCheckEndpointDisplayName = "Health checks";
+        public const string HealthCheckEndpointDisplayName = "Health checks";
 
-        internal static LogEventLevel GetLevel(HttpContext httpContext, double elapsed, Exception exception)
+        public static LogEventLevel GetLevel(HttpContext httpContext, double elapsed, Exception exception)
```

Comment:
Do we need to change these from `internal` to `public`?

Reply:
I was getting error during implementaion but resolved with changes in `*.csproj` changes, so reverting back to `internal`